Documentation/DocuBlocks/Rest/Indexes/post_api_index_inverted.md

```diff
@@ -0,0 +1,371 @@
+indexed as a whole, but only `geopoint` and `aql` Analyzers accept array inputs.
+You cannot use an array expansion if `searchField` is enabled.
+
+@RESTBODYPARAM{storedValues,array,optional,post_api_index_inverted_storedvalues}
+The optional `storedValues` attribute can contain an array of paths to additional 
+attributes to store in the index. These additional attributes cannot be used for
+index lookups or for sorting, but they can be used for projections. This allows an
+index to fully cover more queries and avoid extra document lookups.
+
+@RESTSTRUCT{fields,post_api_index_inverted_storedvalues,array,required,string}
+A list of attribute paths. The `.` character denotes nested fields.
+
+@RESTSTRUCT{compression,post_api_index_inverted_storedvalues,string,optional,}
+Defines how to compress the attribute values. Possible values:
+- `"lz4"` (default): use LZ4 fast compression.
+- `"none"`: disable compression to trade space for speed.
+
+@RESTBODYPARAM{primarySort,object,optional,post_api_index_inverted_primarysort}
+You can define a primary sort order to enable an AQL optimization. If a query
+iterates over all documents of a collection, wants to sort them by attribute values,
+and the (left-most) fields to sort by, as well as their sorting direction, match
+with the `primarySort` definition, then the `SORT` operation is optimized away.
+
+@RESTSTRUCT{fields,post_api_index_inverted_primarysort,array,required,post_api_index_inverted_primarysort_fields}
+An array of the fields to sort the index by and the direction to sort each field in.
+
+@RESTSTRUCT{field,post_api_index_inverted_primarysort_fields,string,required,}
+An attribute path. The `.` character denotes nested fields.
+
+@RESTSTRUCT{direction,post_api_index_inverted_primarysort_fields,string,required,}
+The sorting direction. Possible values:
+- `"asc` for ascending
+- `"desc"` for descending
+
+@RESTSTRUCT{compression,post_api_index_inverted_primarysort,string,optional,}
+Defines how to compress the primary sort data. Possible values:
+- `"lz4"` (default): use LZ4 fast compression.
+- `"none"`: disable compression to trade space for speed.
+
+@RESTBODYPARAM{analyzer,string,optional,string}
+The name of an Analyzer to use by default. This Analyzer is applied to the
+values of the indexed fields for which you don't define Analyzers explicitly.
+
+@RESTBODYPARAM{features,array,optional,string}
+A list of Analyzer features to use by default. They define what features are
+enabled for the default `analyzer`. Possible features:
+- `"frequency"`
+- `"norm"`
+- `"position"`
+- `"offset"`
+
+The default is an empty array.
+
+@RESTBODYPARAM{includeAllFields,boolean,optional,}
+If set to `true`, then all document attributes are indexed, excluding
+any nested attributes that are configured in the `fields` array.
+The `analyzer` and `features` properties apply to the nested fields.
+
+The default is `false`.
+
+{% hint 'warning' %}
```

Comment:
Doesn't it need @HINTS header?

Reply:
`@HINTS` is only for adding admonitions to the top-level of a Rest DocuBlock. It can't be used inside of a parameter description.

Now that I think about it, `{% hint %}` shouldn't be used at all in Rest DocuBlocks other than in `@HINTS` because it won't get rendered in Swagger UI. For ArangoSearch View links, we don't actually have this hint in the DocuBlock, the DocuBlock merely links to the Manual with this hint, where it's perfectly fine to use it:
- https://www.arangodb.com/docs/stable/http/views-arangosearch.html#create-an-arangosearch-view
- https://www.arangodb.com/docs/stable/arangosearch-views.html#link-properties